frontend/src/routes/(root)/(logged)/scripts/run/[...hash]/+page.svelte

```diff
@@ -43,6 +49,13 @@
 			} else {
 				topHash = undefined
 			}
+
+			job_inputs = await ScriptService.getInputHistory({
+				workspace: $workspaceStore!,
+				hash,
+				perPage: 10
+			})
+
```

Comment:
you can make it a separate call that you call "in parallel" to loadScript since there are no dependencies between the 2